modules/document_repository/templates/menu_document_repository.tpl

```diff
@@ -364,9 +364,7 @@
                                 <input type="text" size = "27" name="version" id="version" class="ui-corner-all form-fields form-control input-sm" />
                             </div>
                         </div>
-                        <input type="hidden" name = "user" id = "user" value = "{$User}">
                         <input type="hidden" name = "action" id = "action" value = "upload">
-                        <input type="hidden" id="MAX_FILE_SIZE" name="MAX_FILE_SIZE" value="100000000" />
```

Comment:
A nice idea but this also isn't anywhere; even if it was this sort of validation on the front-end could easily be spoofed.

We should validate this on the back end at some point in the future.

Reply:
Would it be a good place to add it here given this deletion ??

Reply:
Do we still have a distinction between cleanup and feature pull requests? I agree we should have one but I wanted this PR to simply be an enhancement rather than what is effectively a new feature.

Reply:
I'll create a Redmine ticket for it and issue a fix to the bugfix branch

Reply:
@ridz1208 I believe that this sort of thing is actually handled by the `php.ini` file. I'm not sure if we have any explicit conventions with regard to max file upload size but I believe this is handled there rather than here.

Reply:
@johnsaigle @ridz1208 the MAX_FILE_SIZE is effectively define in the php.ini. It might still be a good idea to check on the client side and report that the file is to big instead of waiting for the server to report and error AFTER downloading 2GB of data. A check could also be implemented on the server side before actually trying to do it. Faster for the user to see the error and prevent clogging the bandwidth.

Reply:
@PapillonMcGill I agree. I'll create a ticket in Redmine.